lib/ansible/modules/network/ios/ios_ping.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ios_ping
+short_description: Tests reachability using ping from IOS switch
+description:
+- Tests reachability using ping from switch to a remote destination.
+author:
+- Jacob McGill (@jmcgill298)
+version_added: '2.4'
+extends_documentation_fragment: ios
+options:
+  count:
+    description:
+    - Number of packets to send.
+    required: false
+    default: 5
+  dest:
+    description:
+    - The IP Address or hostname (resolvable by switch) of the remote node.
+    required: true
+  source:
+    description:
+    - The source IP Address.
+    required: false
+    default: null
+  state:
+    description:
+    - Determines if the expected result is success or fail.
+    choices: [ absent, present ]
+    default: present
+  vrf:
+    description:
+    - The VRF to use for forwarding.
+    required: false
+    default: default
+'''
+
+EXAMPLES = r'''
+- provider:
+    host: "{{ ansible_host }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+
+- name: Test reachability to 10.10.10.10 using default vrf
+  ios_ping:
+    provider: "{{ provider }}"
+    dest: 10.10.10.10
+
+- name: Test reachability to 10.20.20.20 using prod vrf
+  ios_ping:
+    provider: "{{ provider }}"
+    dest: 10.20.20.20
+    vrf: prod
+
+- name: Test unreachability to 10.30.30.30 using default vrf
+  ios_ping:
+    provider: "{{ provider }}"
+    dest: 10.30.30.30
+    state: absent
+
+- name: Test reachability to 10.40.40.40 using prod vrf and setting count and source
+  ios_ping:
+    provider: "{{ provider }}"
+    dest: 10.40.40.40
+    source: loopback0
+    vrf: prod
+    count: 20
+'''
+
+RETURN = '''
+action:
+  description: Show what action has been performed.
+  returned: always
+  type: str
+  sample: "Sending 5, 100-byte ICMP Echos to 10.1.100.1, timeout is 2 seconds:"
+updates:
+  description: Show the command sent.
+  returned: always
+  type: list
+  sample: ["ping vrf prod 10.40.40.40 count 20 source loopback0"]
+count:
+  description: Show amount of packets sent.
+  returned: always
+  type: str
+  sample: "2"
+dest:
+  description: Show the ping destination.
+  returned: always
+  type: str
+  sample: "10.40.40.40"
+rtt:
+  description: Show RTT stats.
+  returned: always
+  type: dict
+  sample: {"avg": "2", "max":"8", "min": "1"}
```

Comment:
These are strings in the return values; are you asking me to convert these to integers in the code and then update the docs to reflect the new type?